include/nameof.hpp

```diff
@@ -1002,11 +1001,29 @@ template <typename E>
   return {}; // Value out of range.
 }
 
+
+template <typename E, typename D = std::decay_t<E>, typename U = std::underlying_type_t<D>>
+[[nodiscard]] constexpr auto nameof_enum_or(E value) noexcept -> detail::enable_if_enum_t<E, string> {
```

Comment:
string return value is not constexpr. Probably here need string_view

Reply:
I thought I had removed the constexpr, oh well it's been closed anyways

Reply:
I will make a separate PR to add a customization option